### PR TITLE
feat: update with latest features and standards

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ If everything looks good, they will merge the code and release a new version whi
 
 <!-- markdown-link-check-enable -->
 
-[pull requests]: https://github.com/mineiros-io/terraform-google-folder/pulls
-[pre-commit-file]: https://github.com/mineiros-io/terraform-google-folder/blob/main/.pre-commit-config.yaml
+[pull requests]: https://github.com/mineiros-io/terraform-module-template/pulls
```
Comment (soerenmartius) [inline CONTRIBUTING.md:91]:
```suggestion
[pull requests]: https://github.com/mineiros-io/terraform-google-folder/pulls
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -88,8 +88,8 @@ If everything looks good, they will merge the code and release a new version whi
...
+[pre-commit-file]: https://github.com/mineiros-io/terraform-module-template/blob/main/.pre-commit-config.yaml
```
Comment (soerenmartius) [inline CONTRIBUTING.md:92]:
```suggestion
[pre-commit-file]: https://github.com/mineiros-io/terraform-google-folder/blob/main/.pre-commit-config.yaml
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Changelog should be updated also

File: CONTRIBUTING.md (hunk 1 above)
Comment (mariux) [reply CONTRIBUTING.md:91]: my bad ;)  will fix

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**